microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -254,6 +254,8 @@ To help us improve Microsoft products and services, diagnostic data is aggregate
 
 For Microsoft Edge on Windows 10 and later, sending Optional diagnostic data is determined by your Windows diagnostic data setting. The setting is reflected in `edge://settings/privacy`. Change the Windows settings by going to **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**. On all other platforms, to control the collection of diagnostic data, go to `edge://settings/privacy` and turn on or off **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports**. The setting is used for all profiles associated with the installation of Microsoft Edge on your device.  The setting is not synced across devices. The setting applies to **InPrivate** browsing and Guest mode. Information about websites you visit is never sent while browsing **InPrivate** or in **Guest** mode. If your device is managed with group policies set by your organization, it is described in `edge://settings/privacy`.
 
+As of March 6 2024, Edge diagnostic data is collected separately from Windows diagnostic data on Windows 10 (version 22H2 and newer) and Windows 11 (version 23H2 and newer) devices in the European Economic Area. For these versions, the setting is in `edge://settings/privacy`.
```

Comment:
```suggestion
As of March 6, 2024, Microsoft Edge diagnostic data is collected separately from Windows diagnostic data on Windows 10 (version 22H2 and newer) and Windows 11 (version 23H2 and newer) devices in the European Economic Area. For these versions, the **Xyz** setting is in `edge://settings/privacy`.
```

Reply:
"rather than being a Windows setting that's reflected there" -> Is this necessary? Seems a bit extra given the paragraph before?

Reply:
might remove.  My goal was to make the paragraph understandable in isolation, without requiring reading the many unstructured paragraphs above it (the h2 section needs h4 subheadings, but out of scope for PR).

Reply:
The way that edge://settings/privacy is used differs in this paragraph vs. the previous paragraph, and therefore, more detail is required than just giving this URL; must differentiate explicitly here how edge://settings/privacy is used.  Thus the added phrase at end of the Suggested Change comment is needed.

Reply:
Suggested Change was approved by A.C. offline, so i'm committing the comment